app/views/rescue_codes/new.html.slim

```diff
@@ -0,0 +1,21 @@
+- title t('titles.rescue_codes')
+
+h1.heading = t('headings.rescue_codes')
+p
+  | Below are a set of rescue codes that we have generated for you.
+    You can use these codes to access your account in case you loose
+    access to your phone. Please download these codes and keep them
+    in a safe place.
```

Comment:
Removing the download option would also eliminate the added complexity in `confirm_two_factor_authenticated`.